app.R

```diff
@@ -73,7 +73,22 @@ plot_chart <- function(chart_data, col, title, show_legend=FALSE) {
 #'
 #' @examples
 #' plot_map(map_data, "World Map")
-plot_map <- function(map_data, title, casetype='confirmed') {
+plot_map <- function(map_data, title, data_mode , casetype='confirmed') {
```

Comment:
1. In R we use **<-** not **=**
2.  The code is not formatted (looking at the parentheses)
3. The code is duplicated so many times:
- Split the title into prefix and suffix
- If casetype ==, prefix...===
- If data_mode == 2, suffix = '(Per 1M)', else ''
- Use paste to concatenate them

You should make the data_mode to be optional too, preferrably default to be the absolute case

Remove the title argument, they is created inside this function.